package.json

```diff
@@ -115,6 +115,7 @@
     "release": "6.3.0",
     "request-promise-core": "1.1.2",
     "rimraf": "2.6.3",
+    "schema-utils": "3.0.0",
```

Comment:
Why do we need to add this dependency to the workspace root?

Reply:
Yarn bug, seems like there might be some versioning issues when using package aliases.